helper.js

```diff
@@ -201,7 +201,7 @@ HELPER.compile = (() => {
 	const Autoprefix = ( file ) => {
 		const data = Fs.readFileSync( file, 'utf-8');
 
-		Postcss([ Autoprefixer ])
+		Postcss([ Autoprefixer({ browsers: ['last 2 versions', 'ie 8', 'ie 9', 'ie 10'] }) ])
```

Comment:
you could reason either way. But I think if you wrap it in to a configurable constant or a browserslist file then users can be opinionated about which browsers their app cares about